a6s-cloud/app/Http/Controllers/AnalysisRequestsController.php

```diff
@@ -71,7 +81,10 @@ public function create(Request $request)
                 $total_tweet = $total_tweet + 1;
                 // ユーザ数をカウントする処理を追加する
 
-                // wordcloud用のテキストファイルを作成
+                // wordcloud用のテキストファイルにtweet データを保存
+                // TODO: a6s-cloud-batch に引数として`$localStoragePath . $tweetsFileForWordcloud` を渡して
+                //       tweet データが保存されているファイルを教えてあげる必要がある
+                $storage->append($tweetsFileForWordcloud, $value->text);
```

Comment:
ここはこちらですかね？localStorageに変更したら動きましたー
```suggestion
                $localStorage->append($tweetsFileForWordcloud, $value->text);
```